tutorials/setup.md

```diff
@@ -17,7 +17,7 @@ Latest version of `runc` is expected to be installed on the system. It is picked
 
 **Required**
 
-Fedora, CentOS, RHEL, and related distributions:
+Fedora, RHEL, CentOS and related distributions:
```

Comment:
We may want to make a note here Fedora, CentOS, RHEL <= 7 so RHEL 8 users keep reading down to the dedicated section